src/screens/create-skill/styles.ts

```diff
@@ -0,0 +1,10 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  buttons: {
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+  },
```

Comment:
Check how buttons look if we switch between screens
![Screenshot 2022-02-14 at 10 35 47](https://user-images.githubusercontent.com/64521658/153828408-d39eb056-e5bc-45af-8cff-402cc3dfad44.png)